keyboards/montsinger/rebound/rev5/keymaps/via/rules.mk

```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+RGBLIGHT_ENABLE = yes
```

Comment:
Number of LEDs isn't defined, so this causes it to error out.

Number of LEDs needs to be defined, or this line remove.